pages/ksphere/kommander/1.0/operations/cloud-providers/configure-on-prem-provider/index.md

```diff
@@ -19,19 +19,18 @@ The following procedure describes how to configure an On-Premises infrastructure
 
 1. Go to **Administration > Infrastructure Providers** and select the **Add Infrastructure Provider** button.
 
-    ![Adding an Infrastructure Provider](/ksphere/kommander/1.1.0-beta/img/empty-infrastructure-providers.png)
+   ![Adding an Infrastructure Provider](/ksphere/kommander/1.1/img/empty-infrastructure-providers.png)
```

Comment:
this is a 1.0 page using a 1.1 image. Intentional? I see a lot of the other 1.0 pages being changed from latest to 1.0

Reply:
no its not, but its not changed in this PR, it was wrong before. we should keep images per version to avoid accidentially breaking other versions as we change things.